build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "nozzle"
 
-version := "0.10.6"
+version := "0.10.7"
```

Comment:
can you please leave this out? I'll take care of bumping the version during the release process